pkgs/by-name/ne/nexusmods-app/update.bash

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -I nixpkgs=./. -i bash -p nix-update
+#!nix-shell -I nixpkgs=./. -i bash -p nix-update nixfmt-rfc-style
 #shellcheck shell=bash
 
 set -o errexit -o nounset -o pipefail
 
 package=nexusmods-app
 nix-update "$package"
 "$(nix-build --attr "$package".fetch-deps --no-out-link)"
```

Comment:
Interesting. Is this something that nix-update maybe should implement?

Reply:
Perhaps. I think it's used commonly throughout dotnet packages, IDK how widespread it is elsewhere.